### PR TITLE
GO 1.7 and reorganize folders under pkg

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # registry.namely.tech/namely/broadway-dev:v5
 # If you change this file, or any of the dependencies, build a new image and
 # increase the version number.
-FROM golang:1.6.0
+FROM golang:1.7
```
Comment (bobbytables) [inline Dockerfile]: 🔥

**Comment by therealplato** [comment]
screen shot 2016-08-11 at 2 20 12 pm

**Comment by macat** [comment]
UPS

**Comment by macat** [comment]
i think i didn't add the cmd package

**Comment by macat** [comment]
@therealplato fixed

**Comment by poopoothegorilla** [comment]
+1

**Comment by therealplato** [comment]
+1

**Pull request merged**